book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require './book'
+
+describe Book do
```

Comment:
Kindly make sure that all test files are placed in a different directory (e.g `/spec`). This will ensure that all test files are organized in one place. Currently, the root directory is clustered with too many files.